e2e/cypress/integration/interactive_dialog/full_dialog_spec.js

```diff
@@ -106,7 +110,13 @@ describe('Interactive Dialog', () => {
                     // * Verify that the suggestion list or autocomplete open up on click of input element
                     cy.wrap($elForm).find('#suggestionList').should('not.be.visible');
                     cy.wrap($elForm).find('input').click();
-                    cy.wrap($elForm).find('#suggestionList').scrollIntoView().should('be.visible').children().should('have.length', optionsLength[element.name]);
+                    cy.wrap($elForm).find('#suggestionList').scrollIntoView().should('be.visible').children().then((el) => {
+                        if (element.name === 'someuserselector' && config.ElasticsearchSettings.EnableIndexing) {
```

Comment:
When ES is enabled, the number of returned values varied depending on when it's indexed vs the new data (e.g. new user, etc). On MySQL, it returns consistently with 25 users.